dual_net.py

```diff
@@ -81,6 +83,8 @@ def bootstrap(self):
             tf.train.Saver().save(sess, self.save_file)
 
     def train(self, tf_records, init_from=None, logdir=None, num_steps=None):
+        def should_log(i):
```

Comment:
but why?

Reply:
I guess my point is, it seems like we should be exporting metrics rather than dumping a bunch of logs.

Reply:
we don't want to actually do these computations every step for millions of steps -- it'd be a lot of wasted work.  So what we 'log' are actually the metrics.

Reply:
It's a bit badly named; the "logger" is really a TFSummaryWriter that dumps summary protos to a log file that tensorboard can then parse and visualize.